comfy_panel/special_games/captain.lua

```diff
@@ -989,6 +1004,13 @@ function Public.draw_captain_player_gui(player)
 	b = frame.add({type = "button", name = "captain_player_want_to_be_captain", caption = "I am willing to be a captain", style = "green_button", tooltip = "The community needs you"})
 	b.style.horizontally_stretchable = true
 
+	-- Add a textbox for the player to enter info for the captains to see when picking
+	l = frame.add({type = "label", name = "captain_player_info_label", caption = "Enter any info you want the captains to see when picking players, i.e. 'I will be on discord. I can threatfarm. I can build lots of power.'"})
+	l.style.single_line = false
+	local textbox = frame.add({type = "textfield", name = "captain_player_info", text = special["player_info"][player.name] or "", tooltip = "Enter any info you want the captains to see when picking players."})
+	textbox.style.horizontally_stretchable = true
+	textbox.style.minimal_width = 400
```

Comment:
```suggestion
	textbox.style.width = 0
```
apparently its the right way to do it because `textfield` have some default width

Reply:
Perfect. Thank you.